proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpMetricsUtilsTest.java

```diff
@@ -860,4 +866,92 @@ public void appliesPreprocessorRules() {
 
     assertAllPointsEqual(expectedPoints, actualPoints);
   }
+
+  @Test
+  public void testUpdateAttrsList() {
+    Resource otelResource =
+        Resource.newBuilder()
+            .addAttributes(OtlpTestHelpers.attribute("application", "some-app-name"))
+            .addAttributes(OtlpTestHelpers.attribute("service.name", "some-service-name"))
+            .addAttributes(OtlpTestHelpers.attribute("shard", "some-shard-name"))
+            .addAttributes(OtlpTestHelpers.attribute("cluster", "some-cluster-name"))
+            .build();
+
+    List<KeyValue> attrList =
+        OtlpMetricsUtils.appTagsFromResourceAttrs(otelResource.getAttributesList());
+    assertEquals("some-app-name", OtlpTraceUtils.getAttrValByKey(attrList, "application"));
+    assertEquals("some-service-name", OtlpTraceUtils.getAttrValByKey(attrList, "service"));
+    assertEquals("some-shard-name", OtlpTraceUtils.getAttrValByKey(attrList, "shard"));
+    assertEquals("some-cluster-name", OtlpTraceUtils.getAttrValByKey(attrList, "cluster"));
+  }
+
+  @Test
+  public void testUpdateAttrsListWithServiceKeyPresent() {
```

Comment:
Same request to update the test name, since we're not updating a list of attrs.

Reply:
done